nlpsandboxclient/cli/community.py

```diff
@@ -54,7 +54,3 @@ def get_dates(output):
     nlp = NlpClient(data_node_endpoint=client.DATA_NODE_ENDPOINT)
     dates = nlp.get_dates()
     utils.stdout_or_json(dates, output)
-
-
-if __name__ == '__main__':
```

Comment:
These aren't necessary unless you plan on doing

```
python nlpsandboxclient/cli/community.py ...
```

Reply:
One advantage of the main command is that someone could test the `community` commands during development without requiring to link them to the main program executable. For example `python community.py get_num_users`. Consider keeping this main and calling from there the local click group.

Reply:
Just saw your comment: yes, that's what I had in mind :)